encryptor/config_loader/consul/consul_cli.go

```diff
@@ -0,0 +1,44 @@
+package consul
+
+import (
+	"flag"
+)
+
+const (
+	defaultConsulConfiPath     = "acra/encryptor_config"
+	consulConnectionStringFlag = "consul_connection_api_string"
+)
+
+// CLIOptions keep command-line options related to Consul encryptor config loader.
+type CLIOptions struct {
+	Address             string
+	EncryptorConfigPath string
+}
+
+// RegisterCLIParametersWithFlagSet look up for consul_connection_api_string, if none exists, consul_connection_api_string and consul_kv_config_path
+// will be added to provided flags.
+func RegisterCLIParametersWithFlagSet(flags *flag.FlagSet, prefix, description string) {
+	if description != "" {
+		description = " (" + description + ")"
+	}
+	// TODO: add tls support
```

Comment:
I think we should not accept pure connections, only tls. At least without mutual auth. Because config file is important and configure security controls.

Reply:
Added possibility to have pure/mTLS/TLS connection as we discussed.

Reply:
hm, maybe we can register tls related flags using new [RegisterTLSArgsForService](https://github.com/cossacklabs/acra/blob/master/network/tls_wrapper.go#L115) method? It will register all flags related to TLS including CRL/OCSP and allow to get TLS config with configured certificate verifier

Reply:
Makes sense!